app/src/main/java/com/hedvig/app/feature/dismissiblepager/DismissiblePager.kt

```diff
@@ -48,7 +48,10 @@ abstract class DismissiblePager : DialogFragment() {
         dismiss()
     }
 
-    open fun onLastPageButton() {}
+    @CallSuper
+    open fun onLastPageButton() {
+        dismiss()
+    }
```

Comment:
Hey, are we sure we want all of the children to inherit this behaviour? Instead of making it abstract and make them decide for themselves if they want to necessarily dismiss.

On the HowClaimsWorkDialog.kt it seems like the last button opens a bottom sheet, and maybe we don't want to dismiss the dialog at the same time. Since if the user clicks that, then clicks somewhere else accidentally dismissing the bottom sheet they will wonder where the dialog has gone. Not sure what the expected behaviour is for sure though.

Reply:
Good catch. Fixed!